app/assets/javascripts/discourse/app/instance-initializers/webview-background.js

```diff
@@ -11,7 +11,7 @@ export default {
     if (caps.isAppWebview) {
       window
         .matchMedia("(prefers-color-scheme: dark)")
-        .addEventListener("change", this.updateAppBackground);
+        .addEventListener("change", this.updateAppBackground.bind(this));
```

Comment:
This probably could use a matching `removeEventListener` in a teardown method, so we don't leak listeners in tests.

Reply:
Sure, will add one. (Tests likely won't hit this condition because we never run tests when `isAppWebview` is true. But no harm in adding cleanup.)